crontabber/tests/test_crontabber.py

```diff
@@ -330,6 +370,11 @@ def test_pop(self):
         popped = self.database.pop('foo', 'default')
         eq_(popped, 'default')
         assert_raises(KeyError, self.database.pop, 'bar')
+        try:
+            del self.database['bar']
+            assert False, "it should have raise a KeyError on del"
+        except KeyError:
+            pass
```

Comment:
`assert_raises` and `self.assertRaises` are similar. The second argument is a callable. In my case `del self.database['bar']` is not a callable.